### DISCUSSION TITLE
Is it possible to call the waternet creator (D-Stability) from code?

### DISCUSSION THREAD

**Discussion by breinbaas** [Deltares/GEOLib#166] [Q&A] [answered]
Hi all, I am wondering if it is possible to start the Waternet creator using Python code. It would be great to be able to adjust the waternet creator settings and determine the new headlines based on the waternet creator code. So far I have not found an option to do this from within d-geolib. If anyone knows if this is possible please let me know.

**Comment by carloslubbers** [comment, marked as answer]
GEOLib currently does not support generating the waternet, sorry! This would require an addition to the D-Stability Console to be able to run the waternet creator as well before it can be added to GEOLib. I've forwarded your interest to the team!

**Reply by RutgerTAUW** [reply]
Are there any updates on the request towards the team? Any news on adding this option?

**Reply by carloslubbers** [reply]
Hi @RutgerTAUW, I'm currently not involved with GEOLib/D-Stability as much, I believe it has not received priority from the team yet. I'd recommend sending a request to product owner [Raymond.vanderMeij@Deltares.nl](mailto:Raymond.vanderMeij@Deltares.nl) for this feature, explaining your possible use-case
